### PR TITLE
Add deal

### PR DESCRIPTION
**Pull request by pingzh**

### CODE REVIEW

**Comment by roboxue** [comment]
Updated some gitignore. Ship when you are ready

**Comment by buy** [comment]
need a rebase

File: Groupon UP/Groupon UP/BrowseConstants.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
```
Comment (buy) [inline Groupon UP/Groupon UP/BrowseConstants.swift]: Should this be `DealDetailsConstants`? (since you renamed the `BrowseView` to `DealDetailsView`)

Comment (pingzh) [inline Groupon UP/Groupon UP/BrowseConstants.swift]: Good call.

**Pull request merged**